### PR TITLE
Tweak site

### PR DESCRIPTION
**Pull request by GovernmentPlates**
This PR aims to tweak a few aspects of the site:
* Added OG Tags
* Added a favicon
* SVG-ize logo and hamburger menu icon
* Fix spacing on mobile navigation bar (see below)
* Update the icon pack to the latest version of F.A. (as well as tweak existing social icons and add a link to the Mastodon account)

Before:
image

After:
image

### CODE REVIEW

File: _layouts/home.html [hunk 1]
```diff
@@ -12,7 +12,7 @@
                 </div>
                 <div class="hero-spotlight-header hero-spotlight-header-home">
                     <h1>The effortless open-source tool for event
-                        organisation, archival and collaboration
+                        organization, archival and collaboration
```
Comment (ThiefMaster) [inline _layouts/home.html:15]: Pretty sure this is intentional since the site uses en_GB (see #18)

File: _includes/head.html [hunk 2]
```diff
@@ -5,6 +5,13 @@
 
     <title>Indico - {% if page.title %}{{ page.title }}{% else %}{{ site.title }}{% endif %}</title>
     <meta name="description" content="{{ site.description }}">
+    <link rel="shortcut icon" type="image/x-icon" href="/img/favicon.ico">
+
+    <meta property="og:url" content="{{ page.url | replace:'index.html','' | prepend: site.baseurl | prepend: site.url }}">
+    <meta property="og:type" content="website">
+    <meta property="og:title" content="{% if page.title %}{{ page.title }}{% else %}{{ site.title }}{% endif %}">
```
Comment (ThiefMaster) [inline _includes/head.html:12]: Something like `{{ page.title or site.title }}` doesn't work?

File: _includes/head.html [hunk 3]
```diff
@@ -5,6 +5,13 @@
 
     <title>Indico - {% if page.title %}{{ page.title }}{% else %}{{ site.title }}{% endif %}</title>
     <meta name="description" content="{{ site.description }}">
+    <link rel="shortcut icon" type="image/x-icon" href="/img/favicon.ico">
+
+    <meta property="og:url" content="{{ page.url | replace:'index.html','' | prepend: site.baseurl | prepend: site.url }}">
```
Comment (ThiefMaster) [inline _includes/head.html:10]: i'd either put a space after every `:` or after none - but not a mix of both (IMHO the no-space version is easier to read)

(also WTF is this weird template engine with such an obnoxious filter syntax?!)

File: _layouts/home.html (hunk 1 above)
Comment (pferreir) [reply _layouts/home.html:15]: Indeed. CERN rule: use "Organization" for CERN, "organisation" for everything else.

Comment (GovernmentPlates) [reply _layouts/home.html:15]: Got it - thanks :)

File: _includes/head.html (hunk 2 above)
Comment (GovernmentPlates) [reply _includes/head.html:12]: Yep, this works - I've also used this on the `<title>`

File: _includes/head.html (hunk 3 above)
Comment (GovernmentPlates) [reply _includes/head.html:10]: This was a 1:1 copy of what we are doing in the canonical section. https://github.com/indico/indico.github.io/blob/6e766680c68b7ebe3fb573539f6fda3b44a3f310/_includes/head.html#L15

But sure, I'll remove the excess spaces.

**Comment by ThiefMaster** [comment]
image

maybe better to use the monochrome version like we do on indico itself? visibility isn't that great on dark browser themes... :/

File: _includes/head.html [hunk 4]
```diff
@@ -3,16 +3,24 @@
     <meta name="viewport" content="width=device-width initial-scale=1, user-scalable=no" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
 
-    <title>Indico - {% if page.title %}{{ page.title }}{% else %}{{ site.title }}{% endif %}</title>
+    <title>Indico - {{ page.title or site.title }}</title>
     <meta name="description" content="{{ site.description }}">
+    <meta author="theme-color" content="#1A4A61">
```
Comment (ThiefMaster) [inline _includes/head.html:8]:
```suggestion
    <meta name="theme-color" content="#1A4A61">
```

**Comment by GovernmentPlates** [comment]
> image
> 
> maybe better to use the monochrome version like we do on indico itself? visibility isn't that great on dark browser themes... :/

How about a selection? The one what we already use is the middle tab.

image

**Comment by ThiefMaster** [comment]
IMHO the left one looks best on your screenshot (but I guess your tab background is lighter?).
The center one is probably the safest choice for any background...

File: _includes/footer.html [hunk 5]
```diff
@@ -49,8 +49,9 @@ <h3 class="footer-col-title">OTHER</h3>
         <div class="col-md-10 col-12">Copyright &copy; 2014–2023 CERN. All rights reserved.</div>
         <div class="col-md-2 col-12">
           <span class="footer-icons">
-            <a class="no-decoration" href="https://twitter.com/getindico"><i class="fa fa-twitter fa-lg"></i></a>
-            <a class="no-decoration" href="https://github.com/indico/indico"><i class="fa fa-github-alt fa-lg"></i></a>
+            <a class="no-decoration" href="https://fosstodon.org/@getindico" target="_blank"><i class="fa fa-mastodon fa-lg"></i></a>
+            <a class="no-decoration" href="https://twitter.com/getindico" target="_blank"><i class="fa fa-x-twitter fa-lg"></i></a>
```
Comment (ThiefMaster) [inline _includes/footer.html:53]:
```suggestion
            <a class="no-decoration" href="https://twitter.com/getindico" target="_blank"><i class="fa fa-twitter fa-lg"></i></a>
```

would be my preference. everyone knows that logo and it's just a tiny piece of resistance against musk's bullXhit rename 💩

Comment (GovernmentPlates) [reply _includes/footer.html]: Added lol

**Pull request merged**